twopence/run

```diff
@@ -355,7 +355,7 @@ def nfs_show_kernel_exports():
 # the exports file, and use exportfs to provide the reader with
 # hints of what acutally *did* get exported.
 ##################################################################
-def nfs_write_exports(server, exports):
+def nfs_write_exports(server, exports, reload_nfs = "YES"):
```

Comment:
Sure Olaf, thx for the hint. Is definitely better with Boolean. 

You mean that when skipping the reload of nfs, we can avoid make journal.success() and skip the flush and so on? thx